website/pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,540 @@
+
+It will also ensure determinism of query responses, a key requirement on The Graph Network.
+
+**Enabling the GraphQL Validations will break some existing queries** sent to The Graph API.
+
+To be compliant with those validations, please follow the migration guide.
+
+> ⚠️ If you do not migrate your queries before the validations are rolled out, they will return errors and possibly break your frontends/clients.
+
+# Migration guide
+
+You can use the CLI migration tool to find any issues in your GraphQL operations and fix them. Alternatively you can update the endpoint of your GraphQL client to use the `https://api-next.thegraph.com/subgraphs/name/$GITHUB_USER/$SUBGRAPH_NAME` endpoint. Testing your queries against this endpoint will help you find the issues in your queries.
+
+> Not all subgraphs will need to be migrated, if you are using [GraphQL ESlint](https://the-guild.dev/graphql/eslint/docs) or [GraphQL Code Generator](https://the-guild.dev/graphql/codegen), they already ensure that your queries are valid.
+
+## Migration CLI tool
+
+**Most of the GraphQL operations errors can be found in your codebase ahead of time.**
+
+For this reason, we provide a smooth experience for validating your GraphQL operations during development or in CI.
+
+[`@graphql-validate/cli`](https://github.com/saihaj/graphql-validate) is a simple CLI tool that helps validate GraphQL operations against a given schema.
+
+### **Getting started**
+
+You can run the tool as follows:
+
+```bash
+npx @graphql-validate/cli -s https://api-next.thegraph.com/subgraphs/name/$GITHUB_USER/$SUBGRAPH_NAME -o *.graphql
+```
+
+**Notes:**
+
+- Set or replace $GITHUB_USER, $SUBGRAPH_NAME with the appropriate values. Like: [`artblocks/art-blocks`](https://api.thegraph.com/subgraphs/name/artblocks/art-blocks)
+- The preview schema URL (https://api-next.thegraph.com/) provided is heavily rate-limited and will be sunset once all users have migrated to the new version. **Do not use it in production.**
+- Operations are identified in files with the following extensions [`.graphql`,](https://www.graphql-tools.com/docs/schema-loading#graphql-file-loader)[`.ts`, `.tsx`, `.js`, `jsx`](https://www.graphql-tools.com/docs/schema-loading#code-file-loader) (`-o` option).
+
+### CLI output
+
+The `[@graphql-validate/cli](https://github.com/saihaj/graphql-validate)` CLI tool will output any GraphQL operations errors as follows:
+
+![Error output from CLI](https://i.imgur.com/x1cBdhq.png)
+
+For each error, you will find a description, file path and position, and a link to a solution example (see the following section).
+
+## Run your local queries against the preview schema
+
+We provide an endpoint `https://api-next.thegraph.com/` that runs a `graph-node` version that has validations turned on.
+
+You can try out queries by sending them to:
+
+- `https://api-next.thegraph.com/subgraphs/id/<Qm...>`
+
+or
+
+- `https://api-next.thegraph.com/subgraphs/<name_of_your_subgraph>`
```

Comment:
I believe this should be `https://api.thegraph.com/subgraphs/name/<GITHUB_USER>/<SUBGRAPH_NAME>`

See ex https://thegraph.com/docs/en/deploying/hosted-service/#from-an-existing-contract